rules/network/initial_access_rdp_remote_desktop_protocol_to_the_internet.toml

```diff
@@ -41,13 +41,13 @@ event.category:(network or network_traffic) and network.transport:tcp and (desti
 [[rule.threat]]
 framework = "MITRE ATT&CK"
 [[rule.threat.technique]]
-id = "T1043"
+id = "T1190"
```

Comment:
same technique name, but the ID changed?

Reply:
The existing ID was wrong